packages/cashier/src/pages/account-transfer/account-transfer.tsx

```diff
@@ -84,12 +82,10 @@ const AccountTransfer = observer(({ onClickDeposit, onClickNotes, onClose, setSi
     if (has_no_accounts_balance) {
         return <NoBalance onClickDeposit={onClickDeposit} />;
     }
-    if (is_transfer_confirm) {
-        return <AccountTransferReceipt onClose={onClose} />;
-    }
```

Comment:
In the code we have never rendered side notes for `<AccountTransferReceipt />` but in production, it shows because of a bug that we don't clear the side notes properly. With the new changes and clearing the side notes properly the bug is fixed but it was requested to keep the side notes for some reason, Hence moving `<AccountTransferReceipt />` into `<AccountTransferForm />` where the side notes are.